### PR TITLE
fix(style-attribute-accessor): handle prop and url

### PR DESCRIPTION
**Pull request by brandonseydel**
# Pull Request

## 📖 Description

Fixing the style attribute accessor

### 🎫 Issues

CSS Properties were getting kebab cased on object/array binding which lost the property name. 
eg --background was getting turned to background

Background - When setting a base64 encoded URL on the background the getStyleTuplesFromString was splitting it incorrectly causing the URL to be chopped off at the first ;

## 📑 Test Plan

Added additional tests to cover some of the cases that failed

### CODE REVIEW

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://60006-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

File: packages/kernel/src/platform.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-explicit-any */
 export const emptyArray: any[] = Object.freeze<any>([]) as any;
+export const customPropertyPrefix: string = '--';
```
Comment (bigopon) [inline packages/kernel/src/platform.ts:3]: I think this is a runtime-html concern, let's move it there?

Comment (brandonseydel) [reply packages/kernel/src/platform.ts:3]: Yea that makes sense. I don't see it re-used anywhere else outside.

Comment (brandonseydel) [reply packages/kernel/src/platform.ts:3]: just localize a const to attribute accessor you think?

**Comment by brandonseydel** [comment]
https://jsben.ch/etLLK

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://60025-121594173-gh.circle-artifacts.com/0/test/js-framework-benchmark/webdriver-ts-results/table.html

**Comment by fkleuver** [comment]
Awesome, ty for fixing this 😃

**Pull request merged**